src/index.html

```diff
@@ -4,13 +4,51 @@
     <meta charset="UTF-8">
     <meta
       name="viewport"
-      content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0"
+      content="width=device-width,
+        user-scalable=no, initial-scale=1.0,
+        maximum-scale=1.0,
+        minimum-scale=1.0"
     >
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Search bar airbnb</title>
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Search bar airbnb</h1>
+    <form
+      class="search-bar big-bar"
+      action="#"
+      method="post"
+      data-qa="big">
+      <label>
+        <img
+          class="big-img"
+          src="./images/Search.svg"
+           alt="search"
+        >
+        <input
+          id="search-b"
+          class="input big-input"
+          type="text"
+          placeholder="Try &quot;Los Angeles&quot;">
+      </label>
+    </form>
+    <form
+      class="search-bar small-bar"
+      action="#"
+      method="post"
+      data-qa="small">
+      <label>
+        <img
+          class="small-img"
+          src="./images/Search.svg"
+          alt="search"
+        >
+        <input
+          id="search-s"
+          class="input small-input"
+          type="text"
+          placeholder="Try &quot;Los Angeles&quot;">
+      </label>
+    </form>
```

Comment:
```suggestion
    <form
      class="search-bar"
      action="#"
      method="post"
      data-qa="big">
      <label>
        <img
          class="img"
          src="./images/Search.svg"
           alt="search"
        >
        <input
          id="search-b"
          class="input"
          type="text"
          placeholder="Try &quot;Los Angeles&quot;">
      </label>
    </form>
    <form
      class="search-bar small-bar"
      action="#"
      method="post"
      data-qa="small">
      <label>
        <img
          class="img small-img"
          src="./images/Search.svg"
          alt="search"
        >
        <input
          id="search-s"
          class="input small-input"
          type="text"
          placeholder="Try &quot;Los Angeles&quot;">
      </label>
    </form>
```

lets make it work like this